mypy/build.py

```diff
@@ -275,9 +275,23 @@ def compute_search_paths(sources: List[BuildSource],
     if alt_lib_path:
         mypypath.insert(0, alt_lib_path)
 
+    package_path = tuple(_get_site_packages_dirs(options.python_executable))
+    for site in package_path:
+        if site in mypypath:
+            print("site-packages is in the MYPYPATH. Please remove it.", file=sys.stderr)
```

Comment:
Since there may be multiple site-packages dirs (else why the outer loop :-) it would be helpful to include the offending site value in the messages.